### PR TITLE
show warning state in tab bar

### PR DESCRIPTION
**Pull request by ckiss**
ref: https://github.com/project-vagabond/covid-green-app/issues/326

### CODE REVIEW

File: src/assets/icons/tab-bar/contact-tracing-alert.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (floridemai) [inline src/assets/icons/tab-bar/contact-tracing-alert.svg]: No svgo on this one?

File: src/components/organisms/tab-bar-bottom.tsx [hunk 2]
```diff
@@ -46,24 +46,32 @@ const ctOffUnselected = (
...
+
 const ctOnSelected = (
   <TabBarIcons.ContactTracing.On width={32} height={24} color={colors.purple} />
 );
 const ctOffSelected = (
   <TabBarIcons.ContactTracing.Off
     width={32}
     height={24}
-    color={colors.darkGray}
+    color={colors.purple}
```
Comment (floridemai) [inline src/components/organisms/tab-bar-bottom.tsx:57]: 👍 👍

**Review by floridemai — APPROVED** [review]
Minor comment

File: src/assets/icons/tab-bar/contact-tracing-alert.svg (hunk 1 above)
Comment (ckiss) [reply src/assets/icons/tab-bar/contact-tracing-alert.svg]: looks like another bunch of svgs where not optimized

**Pull request merged**